carapace-server/src/main/java/org/carapaceproxy/server/RuntimeServerConfiguration.java

```diff
@@ -272,7 +273,7 @@ public void configure(ConfigurationStore properties) throws ConfigurationNotVali
             tryConfigureFilter(i, properties);
         }
 
-        healthProbePeriod = getInt("healthmanager.period", 0, properties);
+        healthProbePeriod = getInt("healthmanager.period", DEFAULT_PERIOD, properties);
```

Comment:
You will break tests in an unpredictable way.
The health manager must stay off in tests